### PR TITLE
product page added

### PR DESCRIPTION
**Pull request by Aleesha-azhar**

### CODE REVIEW

File: data/data.ts [hunk 1]
```diff
@@ -34,11 +34,10 @@ export const pages = [
     text: 'Contact Us',
   },
   {
-    path: '/team',
-    text: 'Meet the team',
```
Comment (shahzaib834) [inline data/data.ts:38]: Add this again please

File: data/features.ts [hunk 2]
```diff
@@ -2,4 +2,5 @@ export const FEATURES = {
   blogs: true,
   testimonials: true,
   subscribe: false,
+  product: false,
```
Comment (shahzaib834) [inline data/features.ts:5]: Can you confirm? I think we want to add this as a feature so it should be true. I am not sure though

File: types/interfaces.ts [hunk 3]
```diff
@@ -20,18 +20,13 @@ export interface ServicePost {
   slug: string
 }
 
-export interface EmployeePost {
```
Comment (shahzaib834) [inline types/interfaces.ts:23]: Kindly add the removed interface again. and keep both. i.e. the new one and old one

File: types/interfaces.ts [hunk 4]
```diff
@@ -78,7 +74,7 @@ export interface ServiceWithContent extends ServicePost {
   content: string
 }
 
-export interface EmployeeWithContent extends EmployeePost {
```
Comment (shahzaib834) [inline types/interfaces.ts:81]: Add both here too

**Review by shahzaib834 — COMMENTED** [review]
Kindly revert the changes you made and keep both the new product page code and old team page code.

**Comment by Aleesha-azhar** [comment]
> Kindly revert the changes you made and keep both the new product page code and old team page code.

added

**Comment by umairwaheed** [comment]
@shahzaib834 can you verify this?

**Review by shahzaib834 — APPROVED** [review]
LGTM.

**Review by shahzaib834 — COMMENTED** [review]
@umairwaheed PR Looks good!

**Pull request merged**